cmd/get_ledger_range_from_times.go

```diff
@@ -77,9 +66,11 @@ var getLedgerRangeFromTimesCmd = &cobra.Command{
 			cmdLogger.Fatal("could not json encode ledger range", err)
 		}
 
-		if !useStdout {
+		if path != "" {
+			outFile := mustOutFile(path)
```

Comment:
This is clever, I like this implementation :)

Reply:
:)